readthedocs/core/static-src/core/js/doc-embed/sponsorship.js

```diff
@@ -155,12 +155,12 @@ function init() {
     $.ajax({
         url: rtd.api_host + "/api/v2/sustainability/data/",
         crossDomain: true,
+        cache: false,
```

Comment:
previously the json_callback think was being random generated so that was avoiding hitting the cache, but now that isn't used, so setting cache to false should to the trick, though we don't cache response from the dashboard :dagger:

Reply:
I have also tested that cookies are being sent for this cross-domain request